Project.toml

```diff
@@ -9,29 +9,28 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
-ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
 NLPModelsJuMP = "792afdf1-32c1-5681-94e0-d7bf7a5df49e"
 ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 
 [compat]
-ADNLPModels = "0.3.1"
-ForwardDiff = "0.10.25"
-JuMP = "0.21.5"
-MathOptInterface = "0.9.22"
-ModelingToolkit = "3.20.1"
-NLPModelsJuMP = "0.8.3"
-ReverseDiff = "1.12.0"
-julia = "^1.6.0"
+ADNLPModels = "0.4.0"
+ForwardDiff = "0.10.32"
+JuMP = "1.1.1"
+MathOptInterface = "1.7.0"
+NLPModelsJuMP = "0.11.1"
+ReverseDiff = "1.14.1"
+Symbolics = "4.10.4"
```

Comment:
here are dependency versions.
I'm not quite sure if I should stay with a singular version for each module, or if I add new versions?